af/fastpath/fastpath/tests/data/http_header_field_manipulation_2.json

```diff
@@ -0,0 +1 @@
+{"test_keys":{"requests":[],"agent":"agent","socksproxy":null},"test_start_time":"2019-10-28 11:56:47","input_hashes":[],"probe_ip":"127.0.0.1","id":"3282669b-2397-4d8f-bad3-9cd81825518e","test_helpers":{"backend":"http:\/\/37.218.247.95:80"},"probe_cc":"BR","test_runtime":1.3552720547,"input":null,"probe_city":null,"probe_asn":"AS28573","annotations":{"platform":"linux"},"software_name":"ooniprobe","software_version":"2.2.0","data_format_version":"0.2.0","report_filename":"2019-10-28\/20191028T115647Z-BR-AS28573-http_header_field_manipulation-20191028T115649Z_AS28573_eIrzDM4njwMjxBi0ODrerI5N03zM7qQoCvl4xpapTccdW0kCRg-0.2.0-probe.json","test_version":"0.2.0","bucket_date":"2019-10-28","test_name":"http_header_field_manipulation","report_id":"20191028T115649Z_AS28573_eIrzDM4njwMjxBi0ODrerI5N03zM7qQoCvl4xpapTccdW0kCRg","measurement_start_time":"2019-10-28 12:14:07","backend_version":null,"options":[]}
```

Comment:
can you indent this measurement so the diff review is easier?

Reply:
Done